contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/GobblinDistcpJob.java

```diff
@@ -0,0 +1,195 @@
+package voldemort.store.readonly.mr.azkaban;
+
+import java.util.Map;
+import java.util.List;
+import java.net.URI;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.permission.FsAction;
+import org.apache.hadoop.fs.permission.FsPermission;
+import org.apache.hadoop.mapred.JobConf;
+
+import gobblin.runtime.api.JobExecutionResult;
+import gobblin.runtime.embedded.EmbeddedGobblin;
+import gobblin.runtime.embedded.EmbeddedGobblinDistcp;
+
+import voldemort.utils.Props;
+import voldemort.utils.logging.PrefixedLogger;
+import voldemort.store.readonly.swapper.AdminStoreSwapper;
+
+import azkaban.jobExecutor.AbstractJob;
+
+
+public class GobblinDistcpJob extends AbstractJob {
+    static String source;
+    private final String destination;
+    private final Props props;
+
+    GobblinDistcpJob(String id, String source, String destination, Props props) {
+        super(id, PrefixedLogger.getLogger(AdminStoreSwapper.class.getName(), destination));
```

Comment:
destination is cluster-name or a fully-qualified url?
If it is a fully-qualified url, I suggest to use cluster-name, otherwise, the log message could be too long to read.